context.js

```diff
@@ -196,5 +196,11 @@ module.exports = {
   getNamespace     : get,
   createNamespace  : create,
   destroyNamespace : destroy,
-  reset            : reset
+  reset            : reset,
+  getSharedNamespace: function () {
+    if (!process.sharedNamespace) {
+      process.sharedNamespace = create('shared_' + Date.now() + '_' + Math.random());
```

Comment:
Instead of generating a random name and leaving things to chance, why not consider doing it right and implementing anonymous namespaces as I requested in #52?